.travis.yml

```diff
@@ -1,27 +0,0 @@
-language: node_js
```

Comment:
README is using a badge to track travis status. Does that have equivalence?

Reply:
Yes. added